GameContent/WPTR.cs

```diff
@@ -18,12 +18,22 @@ public class WPTR
 
         private static UIElement lastElementClicked;
 
-        private static UIParent parent;
+        private struct UIParents
```

Comment:
Can you put all the stuff related to the pause menu into it's own separate class?